jenkins-pipelines/upgrade-no-raft/rolling-upgrade-centos7-no-raft.jenkinsfile

```diff
@@ -0,0 +1,17 @@
+#!groovy
```

Comment:
We said to take the latest in each type, not the earliest.
So I would expect:
CentOS9
Debian11
Ubuntu22

Reply:
i don't remember we said the latest of each, and AFAICT we don't support `CentOS9`

i have started with 1, just to verify the changes work, but it should be very easy to apply to any job we want...
i will applying the opposite of current PR